packages/server/src/accounts-server.ts

```diff
@@ -44,6 +32,7 @@ const defaultOptions = {
 
 export class AccountsServer {
   public options: AccountsServerOptions;
+  public tokenManager: any;
```

Comment:
if you can add real typings there would be great

Reply:
sure ! Should be done now